plugin/ipythonPlugins/src/dist/ipython/ipythonPlugin

```diff
@@ -45,4 +45,4 @@ os.environ["PYTHONPATH"] = plugin_dir + old_path
 if os.name == 'nt':
     subprocess.call(["python", plugin_dir+"/ipythonPluginStart.py"], shell=True)
 else:
-    os.execlp("python", "python", plugin_dir+"/ipythonPluginStart.py")
+    os.execlp("/usr/bin/env", "python", "python", plugin_dir+"/ipythonPluginStart.py")
```

Comment:
that's not the shebang i don't see that it would do anything.
it's the 1st line of the file: https://en.wikipedia.org/wiki/Shebang_(Unix)